<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="stateLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnections">
    <property name="title">
     <string>Connections</string>
    </property>
    <addaction name="actionNew_Connection"/>
   </widget>
   <addaction name="menuConnections"/>
  </widget>
  <action name="actionNew_Connection">
   <property name="text">
    <string>New Connection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
